docs/user-guide/nginx-configuration/configmap.md

```diff
@@ -53,9 +53,9 @@ The following table shows a configuration option's name, type, and the default v
 |[ignore-invalid-headers](#ignore-invalid-headers)|bool|true|
 |[retry-non-idempotent](#retry-non-idempotent)|bool|"false"|
 |[error-log-level](#error-log-level)|string|"notice"|
-|[http2-max-field-size](#http2-max-field-size)|string|"4k"|
-|[http2-max-header-size](#http2-max-header-size)|string|"16k"|
-|[http2-max-requests](#http2-max-requests)|int|1000|
+|[http2-max-field-size](#http2-max-field-size) DEPRECATED|string|""|
```

Comment:
Are they deprecated in favor of something else? If so and if we have this directive, we should add it on docs

Reply:
![image](https://github.com/kubernetes/ingress-nginx/assets/5085914/931d88b9-b5c3-41d3-8aff-575165b64b4d)

https://nginx.org/en/docs/http/ngx_http_v2_module.html#http2_max_field_size

Old deprecated directive just took 1 value

```
Syntax: | http2_max_field_size size;
-- | --
http2_max_field_size 4k;
http, server

```
But New Replacement directive takes 2 values 

```
Syntax: | large_client_header_buffers number size;
-- | --
large_client_header_buffers 4 8k;
http, server
```

So I am curious to see how struct is changed to accept 2 values instead of 1. I tried in a branch of mine but I am too dumb so failed

Reply:
we already accept [large_client_header_buffers](https://github.com/kubernetes/ingress-nginx/blob/main/internal/ingress/controller/config/config.go#L264) and [keep_alive_requests](https://github.com/kubernetes/ingress-nginx/blob/main/internal/ingress/controller/config/config.go#L258) from what I can see, I can add this to the docs as well though

Reply:
@rikatz I updated the docs with a notes column that has the info of whats replacing the directives